packages/lesswrong/components/hooks/useNotifyMe.ts

```diff
@@ -96,40 +96,9 @@ export const useNotifyMe = ({
     collectionName: "Subscriptions",
     fragmentName: "SubscriptionState",
     enableTotal: false,
-    skip
+    skip: !currentUser
   });
-
-  if (loading) {
-    return {
-      // Apollo returns `loading: true` when you skip the query.
```

Comment:
The bug is that this isn't consistently correct - sometimes loading is false in this case